controllers/namespacescope_controller.go

```diff
@@ -225,17 +222,10 @@ func (r *NamespaceScopeReconciler) PushRbacToNamespace(instance *operatorv1.Name
 		if toNs == operatorNs {
 			continue
 		}
-
-		if err := r.CreateRole(labels, toNs); err != nil {
-			if errors.IsForbidden(err) {
-				r.Recorder.Eventf(instance, corev1.EventTypeWarning, "Forbidden", "cannot create resource roles in API group rbac.authorization.k8s.io in the namespace %s", toNs)
-			}
+		if err := r.generateRBACForNSS(instance, fromNs, toNs); err != nil {
```

Comment:
Why do you need a separate method generateRBACForNSS for NamespaceScope operator, following method seems has cover this logic

Reply:
For the cluster scope nss operator, it doesn't have a namespace scope role for being copied to the other namespaces. Thus I used another method to create namespace admin roles.

Reply:
I will remove these lines since they don't work in the current case.
```
	// By default, set ibm-namespace-scope-operator service account
	var saNames = []string{"ibm-namespace-scope-operator"}
```